databricks_cli/unity_catalog/cred_cli.py

```diff
@@ -0,0 +1,247 @@
+    Create new data access configuration.
+
+    Calls the 'createDataAccessConfiguration' RPC endpoint of the Unity Catalog service.
+    The specification for the request json can be found at <insert doc link here>.
+    Returns the properties of the newly-created DAC.
+
+    """
+    json_cli_base(json_file, json,
+                  lambda json: UnityCatalogApi(api_client).create_dac(metastore_id, json),
+                  encode_utf8=True)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='List data access configurations.')
+@click.option('--metastore-id', required=True, type=MetastoreIdClickType(),
+              help='Unique identifier of the metastore parent of the DAC(s).')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def list_dacs_cli(api_client, metastore_id):
+    """
+    List data access configurations.
+
+    Calls the 'listDataAccessConfigurations' RPC endpoint of the Unity Catalog service.
+    Returns array of DataAccessConfigurations.
+
+    """
+    dacs_json = UnityCatalogApi(api_client).list_dacs(metastore_id)
+    click.echo(mc_pretty_format(dacs_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Get data access configuration.')
+@click.option('--metastore-id', required=True, type=MetastoreIdClickType(),
+              help='Unique identifier of the metastore parent of the DAC.')
+@click.option('--dac-id', required=True, type=DacIdClickType(),
+              help='Data access configuration ID.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def get_dac_cli(api_client, metastore_id, dac_id):
+    """
+    Get data access configuration details.
+
+    Calls the 'getDataAccessConfiguration' RPC endpoint of the Unity Catalog service.
+    Returns details of the DAC specified by its id (TODO: lookup by DAC name?).
+
+    """
+    dac_json = UnityCatalogApi(api_client).get_dac(metastore_id, dac_id)
+    click.echo(mc_pretty_format(dac_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Delete data access configuration.')
+@click.option('--metastore-id', required=True, type=MetastoreIdClickType(),
+              help='Unique identifier of the metastore parent of the DAC.')
+@click.option('--dac-id', required=True, type=DacIdClickType(),
+              help='Data access configuration ID.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def delete_dac_cli(api_client, metastore_id, dac_id):
+    """
+    Delete data access configuration details.
+
+    Calls the 'deleteDataAccessConfiguration' RPC endpoint of the Unity Catalog service.
+
+    """
+    UnityCatalogApi(api_client).delete_dac(metastore_id, dac_id)
+
+
+def register_cred_commands(cmd_group):
+    # Storage Credential cmds:
+    cmd_group.add_command(create_credential_cli, name='create-storage-credential')
+    cmd_group.add_command(list_credentials_cli, name='list-storage-credentials')
```

Comment:
consistency-wise, name should be `list`, e.g. full command will be `databricks storage-credential list`

Reply:
Addressed in #471.